bullet_train-incoming_webhooks/app/controllers/webhooks/incoming/bullet_train_webhooks_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Webhooks::Incoming::BulletTrainWebhooksController < Webhooks::Incoming::WebhooksController
   def create
-    Webhooks::Incoming::BulletTrainWebhook.create(data: JSON.parse(request.body.read)).process_async
+    Webhooks::Incoming::BulletTrainWebhook.create(data: JSON.parse(request.raw_post)).process_async
```

Comment:
For some reason `request.body.read` is returning an empty string `''` which then causes `JSON.parse` to throw an error. Using `raw_post` seems to work. At least for tests...